examples/pde.hpp

```diff
@@ -861,12 +981,15 @@ SPE10Problem::SPE10Problem(const char* permFile, int nDimensions, int spe10_scal
     }
 
     InitGraph();
-    ComputeGraphWeight(unit_weight);
+    // TODO FIXME: unit_weight is the wrong name for what I am doing here
```

Comment:
If we want to return edge weights corresponding to the case of constant 1 permeability, then I think the updated approach in this PR is doing the right thing. Note that in this case the edge weight is not 1, but weighted by geometrical quantities like face area, cell center distance, etc.

In the original examples (`sampler`, `ml-sampler`, `ml-sampler4`), we used a pure graph with edge weights being 1 (geometrical quantities are not considered). So we need to update the results if we want to honor the finite volume discretization.

Reply:
Based on these comments I removed the option, and rebaselined the `sampler` tests. Now all the tests pass. If you're happy with this and think the rebaselining was legitimate, then I think we're all set.